### DISCUSSION TITLE
A way for auto download and install latest version - batch or something...

### DISCUSSION THREAD

**Discussion by dimo70** [tnodir/fort#318] [Q&A] [answered]
Hi is there a script to autodownload latest exe and install it, or update

**Comment by tnodir** [comment, marked as answer]
You can configure to auto download new version and get notification about it. But you have to install it manually anyway.

- Options: Schedule: enable a "Fort Firewall Update Checker" task to run
- Options: Auto Update: set a "Auto-download new version" flag

**Comment by tnodir** [comment]
Implemented in [v3.14.8](https://github.com/tnodir/fort/releases/tag/v3.14.8).
